service-terrain-profile/pom.xml

```
@@ -0,0 +1,183 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>fi.nls.oskari.extras</groupId>
+    <artifactId>service-terrain-profile</artifactId>
+    <version>1.0</version>
+    <packaging>jar</packaging>
+    <name>Terrain Profile</name>
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <jdk.version>1.8</jdk.version>
+        <oskari.version>[2.4.0,)</oskari.version>
```

Lets' bump up the required version:
```suggestion
        <oskari.version>[2.6.0,)</oskari.version>
```